src/main/webapp/app/shared/consistency-check/package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Does this file have any use? Why is it here?

Reply:
No use.. Just an artifact of the component generation I suppose

Has now been removed 👍